ui/pages/settings/experimental-tab/experimental-tab.component.tsx

```diff
@@ -0,0 +1,314 @@
+            setAddSnapAccountEnabled(!value);
+          },
+          toggleContainerDataTestId: 'add-account-snap-toggle-div',
+          toggleDataTestId: 'add-account-snap-toggle-button',
+          toggleOffLabel: t('off'),
+          toggleOnLabel: t('on'),
+        })}
+      </>
+    );
+  }
+  ///: END:ONLY_INCLUDE_IF
+
+  renderToggleRequestQueue() {
+    const { t } = this.context;
+    const { useRequestQueue, setUseRequestQueue } = this.props;
+    return this.renderToggleSection({
+      title: t('toggleRequestQueueField'),
+      description: t('toggleRequestQueueDescription'),
+      toggleValue: useRequestQueue || false,
+      toggleCallback: (value) => setUseRequestQueue(!value),
+      toggleContainerDataTestId: 'experimental-setting-toggle-request-queue',
+      toggleDataTestId: 'experimental-setting-toggle-request-queue',
+      toggleOffLabel: t('toggleRequestQueueOff'),
+      toggleOnLabel: t('toggleRequestQueueOn'),
+    });
+  }
+
+  renderNotificationsToggle() {
+    const { t } = this.context;
+    const { featureNotificationsEnabled, setFeatureNotificationsEnabled } =
+      this.props;
+
+    return this.renderToggleSection({
+      title: t('notificationsFeatureToggle'),
+      description: t('notificationsFeatureToggleDescription'),
+      toggleValue: featureNotificationsEnabled,
+      toggleCallback: (value) => setFeatureNotificationsEnabled(!value),
+      toggleDataTestId: 'toggle-notifications',
+      toggleOffLabel: t('off'),
+      toggleOnLabel: t('on'),
+    });
+  }
+
+  ///: BEGIN:ONLY_INCLUDE_IF(build-flask,build-beta)
+  // We're only setting the code fences here since
+  // we should remove it for the feature release
+  renderBitcoinSupport() {
+    const { t, trackEvent } = this.context;
+    const { bitcoinSupportEnabled, setBitcoinSupportEnabled } = this.props;
+
+    return (
+      <>
+        <Text
+          variant={TextVariant.headingSm}
+          as="h4"
+          color={TextColor.textAlternative}
+          marginBottom={2}
+          fontWeight={FontWeight.Bold}
+        >
+          {t('experimentalBitcoinSectionTitle')}
+        </Text>
+        {this.renderToggleSection({
+          title: t('experimentalBitcoinToggleTitle'),
+          description: t('experimentalBitcoinToggleDescription', [
+            <a
+              key="btc-account-feedback-form__link-text"
+              href=""
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {t('form')}
+            </a>,
+          ]),
+          toggleValue: bitcoinSupportEnabled,
+          toggleCallback: (value) => {
+            trackEvent({
+              event: MetaMetricsEventName.BtcExperimentalToggled,
+              category: MetaMetricsEventCategory.Settings,
+              properties: {
+                enabled: !value,
+              },
+            });
+            setBitcoinSupportEnabled(!value);
+          },
+          toggleDataTestId: 'bitcoin-accounts-toggle',
```

Comment:
Nit: What about naming this `bitcoin-support-toggle` (to match the flag name?)

Reply:
[573a5b5](https://github.com/MetaMask/metamask-extension/pull/25672/commits/573a5b5d532697070be241916f117395cb93b0ea)